patches/linux/yarn-dependencies.patch

```diff
@@ -1,9 +1,16 @@
 diff --git a/build/npm/postinstall.js b/build/npm/postinstall.js
-index d45d5bc..81a6449 100644
+index 88e3c9eb1bb..4cad4f1a577 100644
 --- a/build/npm/postinstall.js
 +++ b/build/npm/postinstall.js
-@@ -56,3 +56,3 @@ function yarnInstall(dir, opts) {
+@@ -53,8 +53,10 @@ function npmInstall(dir, opts) {
+ 		log(dir, `Installing dependencies inside container ${process.env['VSCODE_REMOTE_DEPENDENCIES_CONTAINER_NAME']}...`);
```

Comment:
Use the command `./patch.sh linux/yarn-dependencies.patch` to edit that file. It will remove the extra lines.